### DISCUSSION TITLE
How to align label element of form input?

### DISCUSSION THREAD

**Discussion by tony9527167** [mantinedev/mantine#4342] [Q&A] [answered]
As mantine doc [said](https://mantine.dev/core/input/#usage) all form input are align to top:

![WX20230601-161223@2x](https://github.com/mantinedev/mantine/assets/63030915/ebd8a85e-dffc-485b-848e-b3061314869d)

how to set it align to left?

![WX20230601-161234@2x](https://github.com/mantinedev/mantine/assets/63030915/d1f8cde4-efeb-4588-b2b3-cae6c08b32bb)

see also https://element-plus.org/en-US/component/form.html#basic-form

**Comment by rtivital** [comment, marked as answer]
Use `Input.Label` component to display label anywhere you need if default layout does not fit your requirements.
